xtask-wasm-run-example/src/lib.rs

```diff
@@ -36,6 +36,14 @@ use syn::{parse, parse_macro_input};
 ///     ```console
 ///     cargo run --example my_example
 ///     ```
+///
+/// ## Arguments
+///
+/// You can give arguments to the macro to customize the example:
+///
+/// * `index` - Content of a custom index.html.
+/// * `static_dir` - Path to a custom static directory.
+/// * `app_name` - If a custom index or static directory is provided, change the app name.
```

Comment:
It always change the app name, no matter if a custom index or static directory is provided.

I think it is not useful to be too specific what this is used for.
```suggestion
/// * `app_name` - Change the app name.
```